suite-native/screen-overlay/src/hooks/useTakeScreenshot.ts

```diff
@@ -0,0 +1,9 @@
+import { useContext } from 'react';
+
+import { ScreenshotContext } from '../components/ScreenshotProvider';
+
+export const useTakeScreenshot = () => {
+    const { takeScreenshot } = useContext(ScreenshotContext);
+
+    return takeScreenshot;
+};
```

Comment:
I divided that because typically you want to access the `screenshot` and `takeScreenshot` properties separately in different files and don't need to import both of these properties.

Also the `useScreenshot` is used only in the package where it is implemented, so like this, we can export only the `useTakeScreenshot` hook.

Reply:
Yeah, but you can just destructure it and take what you need. I mean it's just a nit, but to me it would make sense to have one hook where you just take what you need and can do both. If there ever was a component when you need both, you would have to do that on 2 lines instead of 1. Keep it simple IMO

Reply:
Since the latest mixup refactor, these hooks are not needed anymore. bceb027